docs/user_guide/remote_processing.rst

```diff
@@ -7,98 +7,63 @@ The SiliconCompiler project supports a remote processing model that leverages th
-     - None
-   * - ``checkonly``
-     - None
-   * - ``eda, ..., variable, ...``
-     - None
+.. note::
+
+    Note that our public beta currently only supports open-source tools and PDKs. You can access the public beta without a signup or login, and it is designed to delete your data after your jobs finish, but it is not intended to process proprietary or restricted intellectual property! Please `review our terms of service <https://www.siliconcompiler.com/terms-of-service>`_, and do not submit IP which you are not allowed to distribute.
+
+    Currently, our public beta servers will only return a report containing a rendering and metrics for your build results. For the full GDS results, you can build and install the open-source tools and run the job on your local machine. We provide `setup scripts to help install these tools on Ubuntu systems <https://github.com/siliconcompiler/siliconcompiler/tree/main/setup>`_.
+
+    In the event that our servers are busy processing a large number of jobs, your job may get queued and experience delays in processing. Please be patient during this public beta period.
+
+Even though our publicly-available servers only support open-source IP and tools, the remote API is capable of supporting any SiliconCompiler modules which the server operators wish to install, and we provide a minimal example development server which can be used as a starting point for custom server implementations. You can also find descriptions of the core remote API calls in the :ref:`remote API <Server API>` section.
+
+For a quickstart guide, see the :ref:`Quickstart guide` section.
+
+Configuring a Different Remote Server
+-------------------------------------
+
+You can run the :ref:`sc-configure` command to configure your SiliconCompiler installation with a custom remote endpoint, if you have one. If your remote server requires authentication, you can run the utility with no arguments and fill in the address, username, and password fields that it prompts you for. If your server does not require authentication, you can simply pass its address in as a command-line argument:
+
+``sc-configure https://server.siliconcompiler.com``
+
+If a previous credentials file already exists, you will be prompted to overwrite it. Your credentials file will be placed in ``$HOME/.sc/``, if you want to back it up or delete it.
+
+SiliconCompiler will default to using our public beta address if you have not configured anything, and it will remind you that your design is being uploaded to a public service for processing before starting each remote job.
+
+SiliconCompiler also supports private servers which require authentication to access. If you have such a server to connect to, you will need a credentials text file located at ~/.sc/credentials on Linux or macOS, or at C:\\Users\\USERNAME\\.sc\\credentials on Windows. The credentials file is a JSON formatted file containing information about the remote server address, username, and password.
+
```

Comment:
I thinks some rewording is needed in the above section to make the seams of copy/pasting less visible.

A suggestion:

SiliconCompiler supports running with custom remote endpoints. SiliconCompiler will default to using our public beta address if you have not configured anything, but you can use the :ref:sc-configure command to configure your SiliconCompiler installation on a custom public or private server.

Public Server
^^^^^^^^^^^^^^
If you wish to configure SiliconCompiler for a custom public server, you can simply pass its address in as a command-line argument:
``sc-configure https://server.siliconcompiler.com``

If a previous credentials file already exists, you will be prompted to overwrite it. Your credentials file will be placed in $HOME/.sc/, if you want to back it up or delete it.

Private Server
^^^^^^^^^^^^^^^
If you wish to configure SiliconCompiler for a custom private server which requires authentication, you will need a credentials text file located at ``~/.sc/credentials`` on Linux or macOS, or at ``C:\\Users\\USERNAME\\.sc\\credentials`` on Windows. The credentials file is a JSON formatted file containing information about the remote server address, username, and password.

Reply:
I concur